core/src/test/java/org/infinispan/test/fwk/JGroupsConfigBuilder.java

```diff
@@ -66,8 +67,13 @@ protected Integer initialValue() {
    private static final ConcurrentMap<String, Integer> testUdpIndex = new ConcurrentHashMap<>();
 
    static {
-      JGROUPS_STACK = LegacyKeySupportSystemProperties.getProperty("infinispan.cluster.stack", "infinispan.test.jgroups.protocol", "udp");
+      JGROUPS_STACK = LegacyKeySupportSystemProperties.getProperty("infinispan.cluster.stack", "infinispan.test.jgroups.protocol", "test-udp");
       System.out.println("Transport protocol stack used = " + JGROUPS_STACK);
+      // Load the XML just once
+      protocolStackConfigurator.put("test-tcp", loadProtocolStack("stacks/tcp.xml"));
+      protocolStackConfigurator.put("test-udp", loadProtocolStack("stacks/udp.xml"));
+      protocolStackConfigurator.put("tcp", loadProtocolStack("default-configs/default-jgroups-tcp.xml"));
+      protocolStackConfigurator.put("udp", loadProtocolStack("default-configs/default-jgroups-udp.xml"));
```

Comment:
I'd like to load the stacks lazily, it may be too much work to parse the production stacks every time a developer runs a single test in the IDE. But we can do that later.

Reply:
Once merged, I will create a JIRA for that.